docs/content/configuration/csi_ug_config_create_secret_topology.md

```diff
@@ -0,0 +1,46 @@
+# Creating a Secret with topology awareness
+
+Create an array secret YAML file in order to define the storage credentials (username and password) and address. Use this information for creating a Secret that is topology aware.
+
+**Note:** If you are not using the CSI Topology feature, follow the steps in [Creating a Secret](csi_ug_config_create_secret.md).
+
+Use the Secret file to connect a worker node to a storage system.
+
+1. Create the secret file, similar to the following demo-secret-config.json:
+
+    The `management_address` field can contain more than one address, with each value separated by a comma.
+
+    ```
+        {
+     "demo-system-id-1": {
+       "username": "demo-username-1",
+       "password": "demo-password-1",
+        "management_address": "demo-management-address-1",
+        "supported_topologies": [
+         {
+           "topology.block.csi.ibm.com/demo-region": "demo-region-1",
+           "topology.block.csi.ibm.com/demo-zone": "demo-zone-1"
+         }
+       ]
+     },
+     "demo-system-id-2": {
+       "username": "demo-username-2",
+       "password": "demo-password-2",
+       "management_address": "demo-management-address-2",
+       "supported_topologies": [
+         {
+           "topology.block.csi.ibm.com/demo-region": "demo-region-2",
+           "topology.block.csi.ibm.com/demo-zone": "demo-zone-2"
+         }
+       ]
+     }
+   }
+     ```
+       
+2. Apply the secret using the following command:
+
+    `kubectl create secret generic <secret name> -n <secret namespace> --from-file=config=demo-secret-config.json`
+    
+
+     The `secret/<secret name> created` message is emitted.
```

Comment:
I think a hyphen is missing here

Reply:
@lisyukanna  does the name have to be one word / hyphenated? or can it be more than that? if yes, i'll update in all relevant places

Reply:
as far as I know, yes it should be one word, @oriyarde ?

Reply:
don't know about `have to`, but that is the convention I know.
and regardless, I prefer the actual name, e.g. `demo-secret`

Reply:
I updated a few emitted messages so that there was more consistency.
I also updated here the secret-name with a hyphen and added the same to the command above as well.
DONE